action.yml

```
@@ -19,6 +19,10 @@ inputs:
         description: "Set to true to disable the generation of Job Summaries."
         default: "false"
         required: false
+    disable-auto-build-publish:
+        description: "By default, if the workflow doesn't include at least one 'jf rt build-publish' command, the build-info is published automatically to Artifactory at the end of the workflow. Set to true to disable the automatic build-info publish."
```

```suggestion
        description: "By default, if the workflow ends with collected build-info that was not published using the  'jf rt build-publish' command, the build-info will be automatically published to Artifactory at the end of the workflow. Set to true to disable the automatic build-info publish."
```
Just a suggestion, feel free to modify